tools/yaml-templates/android-test.yml

```diff
@@ -49,6 +49,8 @@ steps:
   - bash: 'chmod u+x install_emulator.sh && ./install_emulator.sh'
     displayName: 'Install Emulator'
     workingDirectory: '$(Agent.BuildDirectory)/androidHost/devtools/ci'
+    timeoutInMinutes: 2
+    retryCountOnTaskFailure: 1
```

Comment:
Is it worth adding a comment here that explains why these are set this way? E.g., "We were seeing failures where blah blah 60 minutes and in our experience 2 minutes was blah blah so lets fail early blah blah blah"

Reply:
(I can see someone coming through here in a year and being like "Why is this set to 2 minutes?"

Reply:
Yeah I like that idea. The real timeout I'm going to go with is 10, but I set it to 2 so I could see what would happen it times out. (The answer is it doesn't actually retry the task, so I think I'm going to try to write a specific recovery task that runs when this task fails)

Reply:
@jekloudaMSFT ADO treats timeout differently from failure btw.